common/models/access-token.js

```diff
@@ -132,26 +137,26 @@ module.exports = function(AccessToken) {
    * @param {Boolean} isValid
    */
 
-  AccessToken.prototype.validate = function(cb) {
+  AccessToken.validate = function(token, cb) {
```

Comment:
This is a breaking change, we need to add a backwards-compatible wrapper method:

```js
AccessToken.prototype.validate = function(cb) {
  AccessToken.validate(this, cb); // or this.toObject()?
};
```